examples/sinus_fit/sinus_fit.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+    Copyright (c) 2018, Etienne de Foras and the respective contributors
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+
+// this example show how to use tiny-dnn library to fit data, by learning a
+// sinus function
```

Comment:
I've heard adding a period at the end of sentence is preferable or Brits will kidnap you and you'll end up having their food for the rest of your entire life.